staging/src/k8s.io/apiserver/pkg/cel/library/cost.go

```diff
@@ -219,7 +200,7 @@ func (l *CostEstimator) CallCost(function, overloadId string, args []ref.Val, re
 		}
 	case "validate":
 		if len(args) >= 2 {
-			format, isFormat := args[0].Value().(*cel.Format)
+			format, isFormat := args[0].Value().(cel.Format)
 			if isFormat {
```

Comment:
I don't know how to check that all instances of this as a pointer got converted correctly... did you check that nothing is hitting this code path with `*cel.Format` pointers any more? (temporarily panic in the !isFormat case, etc)

Reply:
Yeah, I did a few different searches.  Test coverage was good on this one. I wasn't able to get a clean run until I found them all.